session/Concept.proto

```diff
@@ -650,6 +646,7 @@ message Relation {
         message Req {}
         message Iter {
             int32 id = 1;
```

Comment:
Are we meant to keep this? It should be deleted like the rest, right?

Reply:
Ah yes, good catch.